src/components/app/landing/login/LoginModal.tsx

```diff
@@ -0,0 +1,82 @@
+import { Form, Input, Button, Modal, notification } from 'antd';
+import classes from './LoginModal.module.css';
+import Credential from '../../../../types/user/credentials';
+import { useState, useEffect } from 'react';
+
+interface Props {
+  isOpen: boolean;
+  onClose: () => void;
+}
+
+const LoginModal = (props: Props) => {
+  const [formIsValid, setFormIsValid] = useState<boolean>(false);
+  const [enteredEmail, setEnteredEmail] = useState<string>('');
+  const [enteredPassword, setEnteredPassword] = useState<string>('');
```

Comment:
@aritrasen12345 When using antd Form component we do not need to maintain the state if we are not populating the form because on onFinish event we get the form details.